en/lessons/libraries/nimble-publisher.md

```diff
@@ -29,14 +29,20 @@ Now, let's add some posts. We need to start creating a directory that will conta
 /priv/posts/YEAR/MONTH-DAY-ID.md
 ```
 
-For example, we start with these two posts:
+So create these folders:
```

Comment:
Should say `create this folder` - singular since there is only one folder

Reply:
@romenigld thanks but I think that these instructions are too low-level. The reader doesn't need to be told to create each folder, she should already know that in order to create the files, she needs to create the folders that contain them.